charts/kong/example-values/minimal-kong-controller-konnect.yaml

```diff
@@ -0,0 +1,21 @@
+# Basic values.yaml configuration for Kong for Kubernetes with the ingress controller
+# and the controller's Konnect integration enabled.
+
+image:
+  repository: kong
+  tag: "3.1"
+
+env:
+  database: "off"
+
+ingressController:
+  enabled: true
+
+  image:
+    repository: kong/nightly-ingress-controller
+    tag: "2023-02-15"
+    effectiveSemver: "2.9.0"
```

Comment:
We expect to not require this image for actual release, correct?  We can push examples that aren't actually viable yet with the expectation that they won't be properly live until an actual KIC release to the chart, since chart main doesn't become live until we bump the actual Chart.yaml version and release staged changes alongside KIC/Kong version bumps.

```suggestion
```

If we know chart changes require as-yet unreleased images, we should commit for the expected future and demonstrate viability through supplemental PR test documentation (i.e. this isn't something we can test with normal CI, but here's a modified local run demonstrating how it works with as-yet unreleased components).